src/flamegraph/mod.rs

```diff
@@ -9,6 +9,7 @@ use quick_xml::{
     events::{BytesEnd, BytesStart, BytesText, Event},
     Writer,
 };
+use std::default::Default;
```

Comment:
You don't need this `use`; `Default` is in the Rust prelude :)

Reply:
Done